### ISSUE TITLE
[BUG] Mod won't load under SteamPlay (Proton)

### ISSUE THREAD

**Issue by cousteaulecommandant** [valheimPlus/ValheimPlus#783]
- **OS:** Proton Experimental (7.0) / Steam / Ubuntu 18.04 
- **Valheim Game Version:** 0.213.4
- **V+ Mod Version:** 0.9.9.11

**Describe the bug**

***Backstory:*** For some reason, the native Linux version of Valheim no longer runs on my computer, so I'm forced to use the Windows version using the compatibility mode (aka Steam Play aka Proton).  The Windows version seems to work, so now I'm trying to have V+ working on it. ***End of backstory.***

After installing the mod as per the instructions (using WindowsClient.zip instead of UnixServer.zip, obviously), when I launch Valheim I only see "Valheim Mistlands" (not "Valheim Plus") and the bottom right of the screen only displays the Valheim version and not the V+ version.
When I try to connect to a server that has V+, I get an error saying "Incompatible Version"; I understand this is because my game doesn't have V+.

Do you think it's possible to make V+ work under Proton?  (Sorry if I'm asking too much.)

**To Reproduce**

1. Get a Linux machine with Steam installed.
2. Install Valheim and under Properties > Compatibility, enable Steam Play compatibility and select "Proton experimental" (or any recent Proton version).
3. Install Valheim Plus using the WindowsClient.zip archive. _(I assume this is the version I have to install, not the Linux one, right?)_
4. Launch the game.

**Expected behavior**

- Valheim launches with Valheim Plus enabled.
- The logo says "Valheim Plus" (if I recall correctly; at least this was the case the last time I played).
- The bottom right of the screen displays "version" (the Valheim version) and "ValheimPlus" (the V+ version).
- I can connect to servers running V+.

**Actual behavior**

- Valheim launches without Valheim Plus enabled.
- The logo says "Valheim Mistlands".
- The bottom right of the screen displays only "version" (the Valheim version), but not "ValheimPlus" (the V+ version).
-  I cannot connect to servers running V+.

**Troubleshooting**

I tried uninstalling the game, deleting the `.../SteamLibrary/steamapps/common/Valheim/` folder, reinstalling the game (which re-creates that folder), and then installing V+ by unzipping it inside that folder.  It didn't work.

Unfortunately, running V+ in a normal native Linux install isn't an option for me since that version doesn't seem to work at all, with or without mods.

PS: I know I should try to get the Linux version fixed, but that's a side quest.

**Comment by stolenvw** [comment]
i would start here 
https://docs.bepinex.dev/articles/advanced/proton_wine.html

**Comment by cousteaulecommandant** [comment]
> i would start here https://docs.bepinex.dev/articles/advanced/proton_wine.html

...well, that was easy!

All I needed to do was open `winecfg` on the wineprefix of the game, and add the `winhttp` library.
Since I don't have protontricks installed, I had to figure out the wineprefix of the game manually, which in my case was `/opt/shared/SteamLibrary/steamapps/compatdata/892970/pfx/`, and then run:
```sh
WINEPREFIX=/opt/shared/SteamLibrary/steamapps/compatdata/892970/pfx  winecfg
```
and add the `winhttp` library in the Libraries tab.
Worked like a charm.

Thank you!
